src/App.css

```diff
@@ -1,29 +1,65 @@
-.app {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 100vh;
+/* stylelint-disable declaration-block-no-shorthand-property-overrides */
+* {
+  padding: 0;
+  margin: 0;
 }
 
 .main {
-  width: 90%;
-  height: 100%;
-  border: 1px solid #ccc;
+  width: 100%;
+  box-shadow: 1px 1px 10px  rgba(204, 204, 204, 0.13);
+  margin-top: 30px;
 }
 
 /* navbar component */
 .navbar {
-  border: 1px solid black;
   display: flex;
   justify-content: center;
```

Comment:
- [OPTIONAL] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}

.flex-center {
  flex-direction: column;
  align-items: center;
}
``` 

- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉🤓

Reply:
I will implement optional suggestions before merge when code is approved.